# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../unison/unison-Linux.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ruN unison-2.9.20/ubase/util.ml unison-2.9.20-avm/ubase/util.ml
--- unison-2.9.20/ubase/util.ml	2002-08-19 23:15:04.000000000 +0200
+++ unison-2.9.20-avm/ubase/util.ml	2003-11-10 22:46:21.000000000 +0100
@@ -217,7 +217,7 @@
     close_in c;
     match l with
       "Darwin" -> true
-    | "linux" | "FreeBSD" | "SunOS" -> false
+    | "Linux" | "FreeBSD" | "SunOS" -> false
     | _ ->
         Printf.printf
          "Warning: 'uname' returned unrecognized result (%s) in Util.isOSX.\n"
